app/controllers/shopify_app/callback_controller.rb

```diff
@@ -24,8 +24,31 @@ def callback
       end
     end
 
+    def jwt_callback
+      if valid_jwt_auth?
+        create_user_session_from_jwt_callback
+        head(:ok)
```

Comment:
Should we use head(:no_content) instead?

Reply:
I'd rather stick with OK because I think we're likely to add some content later, and I don't want apps to break when the status code switches from 204 => 200.